### PR TITLE
KRACOEUS-8755: rice upgrade

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

**Comment by gathreya** [comment]
test this please.

**Comment by gathreya** [comment]
Once tests run fine, please merge this before tonight's env rebuild. We are trying to get this into our env tomorrow.

File: pom.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@
         <org.eclipse.persistence.version>2.5.2</org.eclipse.persistence.version>
         <pdfbox.version>1.8.6</pdfbox.version>
         <quartz.version>1.8.4</quartz.version>
-        <rice.version>2.5.2-r49495</rice.version>
+        <rice.version>2.5.2-20150127-03d707e6da</rice.version>
```
Comment (leoherbie) [inline pom.xml]: should we exclude the oracle specific jar so our oracle profile works correctly?

Comment (gathreya) [inline pom.xml]: Yes, thanks for catching that. Added exclusions.

**Comment by tdurkin** [comment]
Doesn't look like this made it in to the builds today.  I'll merge it and we can organize a restart.

**Pull request merged**